pkgs/applications/misc/tumpa/default.nix

```diff
@@ -1,17 +1,12 @@
 { lib
-, buildPythonPackage
+, python3
 , fetchFromGitHub
-, setuptools
-, pyside2
-, johnnycanencrypt
-, pythonOlder
 , wrapQtAppsHook
 }:
 
-buildPythonPackage rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "tumpa";
   version = "0.1.2";
```

Comment:
This version AFAIK does not work with newer versions of `johnnycanencryt`, we can mark it broken and unbreak it, when https://github.com/kushaldas/tumpa/milestone/2 is completed.

Till then, `Generate -> Generate key` throws this error:
```
The key(s) not found in the keystore.
Traceback (most recent call last):
  File "/nix/store/k18rc9b6pg28xx2phdmr759jjisi248m-tumpa-0.1.2/lib/python3.10/site-packages/tumpasrc/__init__.py", line 480, in generate
    newk = self.ks.create_newkey(
AttributeError: 'KeyStore' object has no attribute 'create_newkey'. Did you mean: 'create_key'?
```